Poker.BE/Poker.BE.Domain.Tests/Game/TurnTests.cs

```diff
@@ -12,122 +12,6 @@ namespace Poker.BE.Domain.Game.Tests
-            //Arrange
-            var player = new Player();
-            var turn = new Turn(player, new Pot(null));
-            Exception expectedException = null;
-
-            //Act
-            try
-            {
-                turn.Call(50);
-            }
-            catch (NotEnoughMoneyException e)
-            {
-                expectedException = e;
-            }
-
-            player.AddMoney(200);
-            turn.Call(50);
-            var res2 = player.Wallet.AmountOfMoney != 150;
-
-            //Assert
-            Assert.AreEqual(expectedException.Message, "Player doesn't have enough money!");
-            Assert.IsFalse(res2);
-        }
-
-        [TestMethod()]
-        public void FoldTest()
-        {
-            //Arrange
-            var player = new Player();
-            var turn = new Turn(player, new Pot(null));
-
-            //Act
-            turn.Fold();
-            var res1 = turn.CurrentPlayer.CurrentState == Player.State.ActiveFolded;
-
-            //Assert
-            Assert.IsTrue(res1);
-        }
-
-        [TestMethod()]
-        public void BetTest()
-        {
-            //Arrange
-            var player = new Player();
-            var turn = new Turn(player, new Pot(null));
-            Exception expectedException = null;
-
-            //Act
-            try
-            {
-                turn.Bet(50);
-            }
-            catch (NotEnoughMoneyException e)
-            {
-                expectedException = e;
-            }
-
-            player.AddMoney(200);
-            turn.Bet(50);
-            var res2 = player.Wallet.AmountOfMoney != 150;
-
-            //Assert
-            Assert.AreEqual(expectedException.Message, "Player doesn't have enough money!");
-            Assert.IsFalse(res2);
-        }
-
-        [TestMethod()]
-        public void RaiseTest()
-        {
-            //Arrange
-            var player = new Player();
-            var turn = new Turn(player, new Pot(null));
-            Exception expectedException = null;
-
-            //Act
-            try
-            {
-                turn.Raise(50);
-            }
-            catch (NotEnoughMoneyException e)
-            {
-                expectedException = e;
-            }
-
-            player.AddMoney(200);
-            turn.Raise(50);
-            var res2 = player.Wallet.AmountOfMoney != 150;
-
-            //Assert
-            Assert.AreEqual(expectedException.Message, "Player doesn't have enough money!");
-            Assert.IsFalse(res2);
-        }
-
-        [TestMethod()]
-        public void AllInTest()
-        {
-            //Arrange
-            var player = new Player();
-            var turn = new Turn(player, new Pot(null));
-
-            //Act
-            player.AddMoney(200);
-            turn.AllIn();
-            var res1 = player.CurrentState == Player.State.ActiveAllIn;
-
-            //Assert
-            Assert.IsTrue(res1);
-        }
+        //DELETE test class??
```

Comment:
all the turn class is really simple. it doesn't even has methods anymore
I don't see any reason to test it...